parity/cli/mod.rs

```diff
@@ -348,6 +348,10 @@ usage! {
 			"--unlock=[ACCOUNTS]",
 			"Unlock ACCOUNTS for the duration of the execution. ACCOUNTS is a comma-delimited list of addresses.",
 
+			ARG arg_enable_signing_queue: (bool) = false, or |c: &Config| c.account.as_ref()?.enable_signing_queue,
+			"--enable-signing-queue=[BOOLEAN]",
+			"Enables the signing queue for external transaction signing either via CLI or personal_unlockAccount, turned off by default.",
```

Comment:
well,  I'm getting confused by `Enables the signing queue for external transaction signing either via CLI or personal_unlockAccount`. 

In other words, if I run `----enable-signing-queue` it will enable the `SigningQueue` if and if  `--unlock` was not entered at startup. Then I need to make a `RPC personal_unlockAccount` to do so otherwise I will not see any error messages at all?

Also, `--lock` will override this CLI arg (`--enable-signing-queue`) ?

Reply:
sorry can you link to the `--lock` cli argument, I can't find it in the code.

Reply:
> In other words, if I run --enable-signing-queue it will enable the SigningQueue if and only if --unlock was not entered at startup

Yeah basically.

> Then I need to make a RPC personal_unlockAccount to do so otherwise I will not see any error messages at all?

Yeah, correct as well.

Reply:
> sorry can you link to the --lock cli argument, I can't find it in the code.

lol, my bad. I meant `--unlock`

Reply:
yeah unlock overrides it.